### PR TITLE
Fixes old nodeJS redirects.

### PR DESCRIPTION
**Pull request by ZachBaird**
Closes Issue #1820  

**What this code accomplishes:**

* Changes casing in the old redirects to find the desired target appropriately.
* Adds an additional redirect to catch an additional case.

### CODE REVIEW

**Comment by U-C-S** [comment]
> Closes Issue #1828

Typo: 1820

File: config/routes/old_route_redirects.rb [hunk 1]
```diff
@@ -19,8 +19,9 @@
       to: redirect('/paths/full-stack-ruby-on-rails/courses/ruby-on-rails/lessons/%{id}')
 
   # redirect old js pathroutes
-  get '/courses/nodeJS', to: redirect('/paths/full-stack-javascript/courses/nodeJS')
-  get '/courses/nodeJS/lessons/:id', to: redirect('/paths/full-stack-javascript/courses/nodeJS/lessons/%{id}')
+  get '/courses/nodeJS', to: redirect('/paths/full-stack-javascript/courses/nodejs')
+  get '/courses/nodejs', to: redirect('/paths/full-stack-javascript/courses/nodejs')
+  get '/courses/nodeJS/lessons/:id', to: redirect('/paths/full-stack-javascript/courses/nodejs/lessons/%{id}')
```
Comment (KevinMulhern) [inline config/routes/old_route_redirects.rb:24]: Hey @ZachBaird, thanks for doing this. This is my fault 🙈  the course name has that capitalisation in the seeds  but rails turns it to lower case with the routes.

We should be able to remove line 22 and change line 24's url to be redirected to use the lowercase nodejs as well.

Comment (ZachBaird) [reply config/routes/old_route_redirects.rb]: Say no more mon ami, it is done!

**Review by KevinMulhern — APPROVED** [review]
LGTM @ZachBaird, thanks for taking the time to do this mate 💪

**Pull request merged**